REST_API.md

```diff
@@ -143,6 +144,7 @@ Fields description:
 
 Field name | Type | Meaning | Required
 ------ | ------ | ------ | ------
+includesPattern | string | Path pattern to which permissions are applied | Y
```

Comment:
@olegmoz I'd suggest to simplify this sentence to "Path patterns to apply permissions to"